packages/web-react/src/components/FileUploader/stories/FileUploaderDismissible.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { SpiritFileUploaderAttachmentProps } from '../../../types';
+import { useFileQueue } from '../useFileQueue';
+import FileUploader from '../FileUploader';
+import FileUploaderInput from '../FileUploaderInput';
+import FileUploaderList from '../FileUploaderList';
+import FileUploaderAttachment from '../FileUploaderAttachment';
+
+const Story: ComponentStory<typeof FileUploader> = (args) => {
+  const { fileQueue, addToQueue, clearQueue, onDismiss } = useFileQueue();
+
+  const attachmentComponent = ({ id, ...props }: SpiritFileUploaderAttachmentProps) => (
+    <FileUploaderAttachment key={id} id={id} {...props} />
+  );
+
+  return (
+    <FileUploader
+      {...args}
+      id="fileUploaderExampleDismissible"
+      onDismiss={onDismiss}
+      fileQueue={fileQueue}
+      addToQueue={addToQueue}
+      clearQueue={clearQueue}
+    >
+      <FileUploaderInput
+        id="fileUploaderExampleDismissibleInput"
+        name="attachments"
+        label="Label"
+        linkText="Upload your file(s)"
+        labelText="or drag and drop here"
+        helperText="Max file size is 10 MB"
+        validationText="Validation message"
+        maxUploadedFiles={3}
```

Comment:
Files should be limited to 3 but I succeeded with selecting 4 files:

<img width="430" alt="image" src="https://github.com/lmc-eu/spirit-design-system/assets/5614085/40db4641-7288-4e32-b609-cfde60ff2328">

Reply:
This is already fixed now